### ISSUE TITLE
Cannot add custom parameters to emit

### ISSUE THREAD

**Issue by abanghendri** [Power-Components/livewire-powergrid#116]
**Describe the bug**
Hi, thanks for awesome package,
but I face the case that need to add custom params for emit, I use this:

```
Button::add('view')
    ->caption('View')
    ->class('btn btn-primary')
    ->emit('Update', [,'mode'=>'update', 'key' => 'id']),
```
but "mode" is return null while key return the right id of data, so How to do this?

**Comment by luanfreitasdev** [comment]
Thank you very much for reporting this issue, it really wasn't possible.

**Comment by luanfreitasdev** [comment]
Fix - [v1.4.5](https://github.com/Power-Components/livewire-powergrid/releases/tag/v1.4.5)
